web/profiles/custom/yalesites_profile/config/sync/core.entity_view_display.node.profile.default.yml

```diff
@@ -97,6 +97,7 @@ third_party_settings:
       - 'Content fields'
       - 'Custom block types'
       - 'Custom blocks'
+      - Devel
```

Comment:
@codechefmarc I'm not sure ow this keeps creeping in. Can you remove this requirements and look for instances that crept into the codebase?